README.md

```diff
@@ -84,114 +53,100 @@ To remove a dependency from your project, you can use the `poetry remove` comman
 poetry remove {package_name}
 ```
 
-## Creating and Using Virtual Environments with Poetry
+## Create and Use Virtual Environments with Poetry
 
-You can create a virtual environment for the project by navigating to the project's root directory and run:
+You can create a virtual environment for the project by navigating to the project's root directory and running the following command:
 
 ```bash
 poetry install
 ```
 
 This will create a new virtual environment and install the project's dependencies.
 
-### Activating the Virtual Environment
+### Activate the Virtual Environment
 
-To activate the virtual environment, use `poetry shell`. This will start a new shell session with the virtual
-environment activated:
+To activate the virtual environment, use `poetry shell`. This will start a new shell session with the virtual environment activated:
```

Comment:
Does procedure contain a sequence of steps (then use an ordered list)? Or are they all alternatives to each other?

```suggestion
To activate the virtual environment, use `poetry shell`. This starts a new shell session with the virtual environment activated:
```

Reply:
@grischperl Can you help with this question?

Reply:
The commands should be run in order. So, first run `poetry install` then run `poetry shell`.

Reply:
No, not really. If you want to, you could formulate in in a way that there are steps:

1. `poetry shell` to activate the environment
2. `poetry run` to execute a command
3. `deactivate` to exit the environment

Reply:
But in my opinion, it's also fine, as it's called "Activate the Virtual Environment" to say the relevant step is `poetry shell`.

And `poetry run` and `deactivate` is additional information

Reply:
Oh, now I get it, thank you :)